config_defaults.yml

```diff
@@ -556,46 +556,54 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 1.0'
-        platforms: ruby
-      - gem: 'puppet-module-posix-dev-r#{minor_version}'
-        version: '~> 1.0'
-        platforms: ruby
-      - gem: 'puppet-module-win-default-r#{minor_version}'
-        version: '~> 1.0'
-        platforms:
-          - mswin
-          - mingw
-          - x64_mingw
-      - gem: 'puppet-module-win-dev-r#{minor_version}'
+      - gem: 'json'
+        version: '~> 2.0'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '~> 3.0'
+      - gem: 'facterdb'
+        version: '~> 1.18'
+      - gem: 'metadata-json-lint'
+        version:
+        - '>= 2.0.2'
+        - '< 4.0.0'
+      - gem: 'puppetlabs_spec_helper'
+        version:
+        - '>= 2.9.0'
```

Comment:
Why 2.9.0? Looking through https://github.com/puppetlabs/puppetlabs_spec_helper/blob/main/CHANGELOG.md I think you at least want 2.15.0 since that works with Ruby 2.7. Then it looks like you depend on the reconfigurability of puppet-lint in `Rakefile.erb` so that's at least 3.0.0.

Reply:
Same as above, was the original pin